### ISSUE TITLE
bot stops playing after around 60 seconds

### ISSUE THREAD

**Issue by Victorce76** [codetheweb/muse#926]
Let me preface this request with saying I by no means have a ton of knowledge about how this works, I simply found this online and taught myself just enough to get this bot working for my personal discord server, it worked without any issues for quite a few months now, but all of a sudden this issue developed. Any help would be appreciated, the server is too quiet without the bot working.

---
name: Support request
about: Something's not working like it should be
title: ''
labels: support
assignees: ''

---

**What's happening?**
When asking the bot to play a song, it initially joins the voice channel and start playing that song, but after about 60 seconds to goes silent while remaining in the channel. if I send the Now-Playing command the timer on the song updates as if it was still playing.

**Logs**
Unsure of where to find these.

```

```

**Screenshots**

**Additional context**

**Runtime**
I'm running Muse:

- [ x] Inside a Docker container (i think? I followed the instructions on how to run with Docker in the README file.)

**Versions**
- Muse: 2.2.2
- Docker (if applicable): 4.17.1
- OS: Windows 10

**Comment by LosBagros** [comment]
same

**Comment by LosBagros** [comment]
log
```
/usr/app/node_modules/ytsr/lib/parseItem.js:355
    .reelPlayerOverlayRenderer.reelPlayerHeaderSupportedRenderers.reelPlayerHeaderRenderer;
                                                                  ^

TypeError: Cannot read properties of undefined (reading 'reelPlayerHeaderRenderer')
    at parseShort (/usr/app/node_modules/ytsr/lib/parseItem.js:355:67)
    at parseItem (/usr/app/node_modules/ytsr/lib/parseItem.js:28:14)
    at <anonymous> (/usr/app/node_modules/ytsr/lib/parseItem.js:409:30)
    at Array.map (<anonymous>)
    at parseShelf (/usr/app/node_modules/ytsr/lib/parseItem.js:409:21)
    at parseItem (/usr/app/node_modules/ytsr/lib/parseItem.js:32:14)
    at catchAndLogFunc (/usr/app/node_modules/ytsr/lib/parseItem.js:88:12)
    at module.exports (/usr/app/node_modules/ytsr/lib/parseItem.js:111:46)
    at <anonymous> (/usr/app/node_modules/ytsr/lib/main.js:53:34)
    at Array.map (<anonymous>)

/********************************************************************************************************************************************************************************************************
failed at func parseItem: Cannot read properties of undefined (reading 'reelPlayerHeaderRenderer')
pls post the the files in /usr/app/node_modules/ytsr/dumps to https://github.com/TimeForANinja/node-ytsr/issues
os: linux-arm64, node.js: v18.7.0, ytsr: 3.8.0
********************************************************************************************************************************************************************************************************\
```

**Comment by MaximusRaeke** [comment]
This is also happening in my Muse Docker container today.

**Comment by codetheweb** [comment]
Hmm, this should have been fixed in v2.2.1 with https://github.com/codetheweb/muse/pull/907. Just tested and can't reproduce with the latest version of Muse.

@fuji97 is this happening for you again too?

**Comment by Victorce76** [comment]
well, after looking at it closer I realized I was actually still running 2.1.7 somehow, but changing the image tag to "2.2.2" instead of "latest" forced it to update to the newer version, and It appears to be working fine now. Sorry for the misinformed post to begin with, I didn't realized I wasn't actually running on the newest version.
